### ISSUE TITLE
Dry run mode

### ISSUE THREAD

**Issue by PerilousApricot** [dmwm/CRABClient#2970]
Hey all,

It would be pretty nice for there to be a client command to just submit one job, sending the output to the same directory the "real" jobs would be sent, but with a different filename (to prevent collisions or to keep from accidentally picking up the output file with an errant wildcard)

Putting on my "doing analysis" hat, I've (recently) gotten burned before where I've run a CMSSW configuration manually and thought everything was okay, then submitted a full dataset of jobs only to find out that I boned a stageout configuration option, forgot to use/update the example files in my PoolSource or screwed up my pyconfig_options so the configuration.

Basically, a way to get test jobs to run in exactly the same environment as how the "real" jobs will run, maybe with the only difference being turning down the jobsplitting to something really low
